cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -369,10 +380,12 @@ Future<Void> rollingUpdateForNewCaKey() {
         }
 
         if (podRollReasons.shouldRestart()) {
+            PemTrustSet clusterCaTrustSet = new PemTrustSet(clusterCa.caCertSecret());
+            PemAuthIdentity coAuthIdentity = PemAuthIdentity.clusterOperator(coSecret);
```

Comment:
could be make explicit that it's about TLS auth from the var name? Something like `coTlsAuthIdentity`? just `AuthIdentity` could be anything, even if we know it's about using TLS between CO and the cluster.
Of course, this comment applies to all places where we have something like this as a variable or a parameter in a method.